spec/peeps_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'peeps'
+require 'database_helpers'
+
+describe Peep do
+  describe '.all' do
+    it 'returns all bookmarks' do
+      connection = PG.connect(dbname: 'chitter_test')
+      peep = Peep.create(message: 'This is a peep!', created_at: 'created_at')
+      peeps = Peep.all
+      expect(peeps.length).to eq 1
+      expect(peeps.first.message).to eq 'This is a peep!'
+    end
+  end
+  describe '.create' do
+    it 'adds a bookmark' do
+      peep = Peep.create(message: 'Test peep', created_at: '2020-03-13 14:23:25.164956')
```

Comment:
I know we did a bit of refactoring together so this probably didn't happen due to time constraints, but now that the `create` method only takes in one argument, this line should be updated to reflect that